mne_bids_pipeline/tests/configs/config_ERP_CORE.py

```diff
@@ -50,7 +49,7 @@
 # Suppress "Data file name in EEG.data (sub-019_task-ERN_eeg.fdt) is incorrect..."
 read_raw_bids_verbose = "error"
 
-eeg_template_montage = mne.channels.make_standard_montage("standard_1005")
+eeg_template_montage = "standard_1005"
```

Comment:
This change would disable testing with a `DigMontage`.

Instead of doing this, I'd suggest to fix the type annotation in our config to accept `str | mne.channels.DigMontage`

Reply:
> This change would disable testing with a DigMontage.

This was never officially documented though. Do we really need to / want to support it?

Reply:
see for example:

https://mne.tools/mne-bids-pipeline/1.4/settings/general.html#mne_bids_pipeline._config.eeg_template_montage

Reply:
I would almost assume that we added this test for a reason ... @agramfort @dengemann Do you know if and why we required setting a montage from a DigMontage instance instead of just passing the template montage name as a string? Was this something you needed at some point?

Reply:
Ahh, doing a bit of `git` digging I can see https://github.com/mne-tools/mne-bids-pipeline/pull/407 by @dengemann so it was indeed intentional... I'll add support back in

Reply:
Great, thanks!